pkg/utils/resources/suite_test.go

```diff
@@ -0,0 +1,393 @@
+/*
```

Comment:
Seems like we are just checking limits across all of these tests. Can we also check resource requests as well? I don't want us to have good logic in our limits logic but fail to catch a miss in our requests calculation logic since we don't have tests for it here.